dotnet-directory

```diff
@@ -11,7 +11,7 @@ print_err()
 }
 
 print_usage() {
-    echo "usage: $0 [--root (default) | --sdk | --framework <runtime version> | --help ]" 1>&2
+    echo "usage: $0 [--root (default) | --sdk | --framework <framework version> <runtime version> | --help ]" 1>&2
```

Comment:
Can you change this to keep `--framework` as it was?
And add `--aspnet` as the argument to get the ASP.NET Core dir?